libr/core/cfile.c

```diff
@@ -1072,15 +1069,21 @@ R_API int r_core_file_list(RCore *core, int mode) {
 		}
 		from = 0LL;
 		switch (mode) {
-		case 'j':
-			r_cons_printf ("{\"raised\":%s,\"fd\":%d,\"uri\":\"%s\",\"from\":%"
-				PFMT64d ",\"writable\":%s,\"size\":%d}%s",
-				r_str_bool (core->io->desc->fd == f->fd),
-				(int) f->fd, desc->uri, (ut64) from,
-				r_str_bool (desc->perm & R_PERM_W),
-				(int) r_io_desc_size (desc),
-				iter->n? ",": "");
+		case 'j': {  // "oij"
+			PJ * pj = pj_new ();
+			pj_o (pj);
+			pj_ks (pj, "raised", r_str_bool (core->io->desc->fd == f->fd));
+			pj_ki (pj, "fd", (int) f->fd);
+			pj_ks (pj, "uri", desc->uri);
+			pj_kn (pj, "from", (ut64) from);
+			pj_ks (pj, "writable", r_str_bool (desc->perm & R_PERM_W));
```

Comment:
writable should be a bool. use pj_kb, not a string

Reply:
Replaced with `pj_kb`